src/modules/Checkbox/Checkbox.js

```diff
@@ -148,8 +148,9 @@ export default class Checkbox extends Component {
     const { id } = this.props
     const { checked, indeterminate } = this.state
 
+    const isLabelClick = _.invoke(this.labelRef.current, 'contains', e.target)
```

Comment:
Previously there was an equality comparison, but in this case `contains()` should be used.